### PR TITLE
Create codeql-analysis.yml

### PR DESCRIPTION
**Pull request by florian-vuillemot**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
```
Comment (florian-vuillemot) [inline .github/workflows/codeql-analysis.yml]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, release/* ]
+  pull_request:
+    branches: [ develop ]
+  schedule:
+    - cron: '22 15 * * 2'
```
Comment (khaitruong922) [inline .github/workflows/codeql-analysis.yml:9]: Is this just an arbitrary time of the day?

**Review by khaitruong922 — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml (hunk 2 above)
Comment (florian-vuillemot) [reply .github/workflows/codeql-analysis.yml:9]: Yes, default provide by Github :)

Comment (florian-vuillemot) [reply .github/workflows/codeql-analysis.yml:9]: And because things are never random https://en.wikipedia.org/wiki/Patch_Tuesday ;)

**Pull request merged**